.github/workflows/scripts/e2e/gmc_gaudi_test.sh

```diff
@@ -10,6 +10,7 @@ MOUNT_DIR=/home/$USER_ID/.cache/huggingface/hub
 IMAGE_REPO=${OPEA_IMAGE_REPO:-""}
 CODEGEN_NAMESPACE="${APP_NAMESPACE}-codegen"
 CODETRANS_NAMESPACE="${APP_NAMESPACE}-codetrans"
+DOCSUM_NAMESPACE="${APP_NAMESPACE}-docsum"
```

Comment:
Make sure you clean up this namespace in function cleanup_gmc(). There are many docsum namespace remained in CI k8s environment. That's not correct.

Reply:
nice catch